### PR TITLE
Time

### PR DESCRIPTION
**Pull request by 1982irushka**
create time formating  and gender validation.

### CODE REVIEW

File: scripts/index.js [hunk 1]
```diff
@@ -55,16 +70,17 @@ newsAllPromise
         url,
         kids,
       } = newsOne;
-
       const commentsHtml = comments
         .filter(({ id }) => (kids ?? []).slice(0, 4).includes(id))
+        .some()
```
Comment (evgeniush) [inline scripts/index.js]: I suppose .some() can be removed

File: scripts/index.js [hunk 2]
```diff
@@ -55,16 +70,17 @@ newsAllPromise
...
+        .map(({ time, ...rest }) => ({ time: timeFormat(time), ...rest }))
```
Comment (evgeniush) [inline scripts/index.js]: ```.map(({ time, ...rest }) => ({ time: timeFormat(time), ...rest }))```
If I'm not mistaken this piece of code was for demonstration purposes. Remove?

File: scripts/index.js [hunk 3]
```diff
@@ -77,7 +93,7 @@ newsAllPromise
       const source = url && hostname ? `<a href="${url}">${hostname}</a>` : '';
 
       return `${accom}
-        <li class="generic-list__item">
+        <li class="generic-list__item">ss
```
Comment (evgeniush) [inline scripts/index.js]: ss

please remove

**Review by evgeniush — CHANGES_REQUESTED** [review]

File: scripts/index.js (hunk 2 above)
Comment (1982irushka) [reply scripts/index.js]: Fixed

File: scripts/index.js (hunk 3 above)
Comment (1982irushka) [reply scripts/index.js]: fixed

Comment (1982irushka) [reply scripts/index.js]: Fixed

**Review by evgeniush — APPROVED** [review]

**Pull request merged**